src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -160,6 +160,9 @@
 --    add media_type table, primary key and fk from media
 --    add is_system_user and account_guid to user_account
 --    add display_name to courthouse
+--v54 add checksum to annotation_document and transcription_document
+--    add display_state to transcription_type and transcription_urgency
+--    add display_name to transcription_status

```

Comment:
Why do we need this? What needs to happen to our https://github.com/hmcts/darts-api/blob/master/src/main/resources/db/reference/modernised_darts_standing_data.sql that keeps getting neglected with new NOT NULL columns

Reply:
Done in commit a6f7b9e9